warehouse/utils/admin_flags.py

```diff
@@ -0,0 +1,33 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sqlalchemy import Column, Boolean, Text
+
+from warehouse import db
+
+
+class AdminFlag(db.Model):
+
+    __tablename__ = "warehouse_admin_flag"
+
+    id = Column(Text, primary_key=True, nullable=False)
+    description = Column(Text, nullable=False)
+    enabled = Column(Boolean)
+
+    @classmethod
+    def is_enabled(cls, session, flag_name):
```

Comment:
This should probably either be a static method, or it should use ``cls`` in the body instead of ``AdminFlag``.

Reply:
yeah, also missed while factoring to a classmethod. do you have a preference between classmethod or static method for this?

Reply:
I don't really. I'd maybe use a classmethod just to make inheritance work... but I don't think that really matters since we're never going to use inheritance here anyways.